### DISCUSSION TITLE
bridge sepolia testnet to zksync sepolia testnet

### DISCUSSION THREAD

**Discussion by owolabi-victor** [Cyfrin/foundry-full-course-cu#2608] [Q&A]
replacement fee too low [ See: https://links.ethers.org/v5-errors-REPLACEMENT_UNDERPRICED ] (error={"details":"replacement transaction underpriced","shortMessage":"An internal error was received.","name":"InternalRpcError","version":"viem@2.7.16","message":"An internal error was received.\n\nDetails: replacement transaction underpriced\nVersion: viem@2.7.16","cause":{"code":-32603,"message":"replacement transaction

i keep getting this error what do i do

**Comment by EngrPips** [comment]
Hello @owolabi-victor, I am unsure what that error is or how to solve it. I will say leave it and proceed. zksync part of the course is optional so don't stress over it.

**Reply by owolabi-victor** [reply]
thank you so much

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
